_data/projects/trapteam-cc.yml

```
@@ -0,0 +1,12 @@
+name: Trap Team CC
+desc: A website and backend service for pest trappers
+site: trapteamnz.tk/git.html
+tags:
+# Note: these are tags categorizing the project, not issue labels.
+- html
+- easy
+- javascript
+- beginners
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/TrapTeamCCNZ/TrapTeamMainSite/labels/up%20for%20grabs
```

This corrects the URL to match the label name (`%20` is an encoded space):

```suggestion
  link: https://github.com/TrapTeamCCNZ/TrapTeamMainSite/labels/up-for-grabs
```